CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 * Support `feesfines` interface version `17.0`. Refs UIU-2248.
 * Update sub permissions in `ui-users.edituserservicepoints` permission set. Fixes UIU-2244.
 * Replace `babel-eslint` with `@babel/eslint-parser`; import global babel config. Refs UIU-2253, UIU-2254.
+* Automatic fees/fines are appearing in New Fee/Fine `Fee/fine type` drop-down. Refs UIU-2411.
```

Comment:
should it be anything like "Fix issue when ..."?

Reply:
I think it could be this way. As I know we do not have any requirements for changelog file, and we can compose by ourselves the changelog entry or I prefer to use JIRA's title.

Reply:
ok. asked just for information